.gitignore

```
@@ -46,6 +46,9 @@ build.ninja
 *_autogen
 /Testing
 
+# Custom user scripts
+*.sh
```

I think this will hide the two scripts we have in the tree, but I may be wrong:
```
./translations/generate_skips.sh
./translations/stats.sh
```
I suggest to restrict it to the root folder,
```suggestion
/*.sh
```